engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -470,11 +477,14 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
                                         failedJobKey: JobKey,
                                         reason: Throwable,
                                         returnCode: Option[Int] = None,
-                                        jobExecutionMap: JobExecutionMap = Map.empty
+                                        jobExecutionMap: JobExecutionMap = Map.empty,
+                                        omitFromWorkflowLevelFailures: Boolean = false
```

Comment:
Nitpick that you can feel free to ignore if you disagree - IMO boolean arguments where `true` represents negation require readers to think too hard. I think it's easier to understand if it's `includeInWorkflowLevelFailures: Boolean = true`.